game_start/game.js

```diff
@@ -32,66 +32,64 @@ const result = 0;
 // logika gry
 
 const showCards = function () {
-	activeCard = this;
-	activeCard.classList.remove('hidden');
-
-	if (activeCards.length === 0) {
-		activeCards[0] = activeCard;
-		return;
```

Comment:
to jest fajne rozwiązanie. Zawsze staramy się jak najszybciej "wyjść z funkcji" tzw. early return.